### PR TITLE
KEYCLOAK-14006 Allow administrator to add additional fields to be fet…

### PR DESCRIPTION
**Pull request by BartoszSiemienczuk**
Allow administrator to add additional fields to be fetched with Facebook profile request.

Added fetchedFields to Facebook Id Provider config which is appended to profile URL to allow fetching additional profile fields. 

Documentation PR: https://github.com/keycloak/keycloak-documentation/pull/932
JIRA: https://issues.redhat.com/browse/KEYCLOAK-14006
Discussion: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/keycloak-dev/trdO_fGJe2w/jKjn0b_zAQAJ

### CODE REVIEW

**Comment by BartoszSiemienczuk** [comment]
This would need a `facebook.profile.birthday` (with a value of test profile birthday) added to social login test properties in order for tests to pass. e.g. `facebook.profile.birthday=06/19/1994`

**Comment by vmuzikar** [comment]
Will try to review by the end of this week.

File: services/src/main/java/org/keycloak/social/facebook/FacebookIdentityProvider.java [hunk 1]
```diff
@@ -18,34 +18,30 @@
 package org.keycloak.social.facebook;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import org.keycloak.OAuthErrorException;
+
+import org.apache.commons.lang.StringUtils;
```
Comment (vmuzikar) [inline services/src/main/java/org/keycloak/social/facebook/FacebookIdentityProvider.java:22]: Just a nitpick. This import seems to be unused.

**Review by vmuzikar — DISMISSED** [review]
@BartoszSiemienczuk Thank you for your contribution! Found one tiny nitpick that is non-blocking, so approving.

**Comment by abstractj** [comment]
@BartoszSiemienczuk would you like to follow up on @vmuzikar's comments so we can merge?

**Comment by BartoszSiemienczuk** [comment]
@abstractj sure I will take care of this in coming hours.

**Comment by BartoszSiemienczuk** [comment]
I've removed the mentioned import and rebased on current upstream master.

**Review by vmuzikar — APPROVED** [review]
@BartoszSiemienczuk Thanks for the update!

**Review by abstractj — APPROVED** [review]

**Pull request merged**